src/display/Drawer/Drawer.tsx

```diff
@@ -6,7 +6,7 @@ import { useIsMdScreen } from "src/guidelines/theme/mediaQueries";
 export type DrawerProps = {
   openFrom?: "left" | "right";
   isOpen: boolean;
-  header?: React.ReactNode;
+  header: React.ReactNode;
```

Comment:
Why did you delete the nullable to the header ? the header is still nullable, right ?